lib/rules/max-nesting-depth/README.md

```diff
@@ -281,14 +305,20 @@ a {
 }
 ```
 
-### `ignoreAtRules: ["/regex/", /regex/, "string"]`
+### `ignoreAtRules`
+
+```json
+{ "ignoreAtRules": ["array", "of", "at-rules", "/regex/"] }
+```
 
 Ignore the specified at-rules.
 
-For example, with `1` and given:
+Given:
 
 ```json
-["/^--my-/", "media"]
+{
+  "max-nesting-depth": [1, { "ignoreAtRules": ["/^--my-/", "media"] }]
+}
```

Comment:
👍  Providing a complete example is much clearer in these instances than the previous "For example, with x and given:"